app/styles/components/_dependency-table.scss

```diff
@@ -0,0 +1,47 @@
+.description {
```

Comment:
`.description` could be a common class name and `hr` will catch all `hr` elements -- can you scope these rules?